python/tests/test_preequilibration.py

```diff
@@ -472,6 +474,55 @@ def test_newton_steadystate_check(preeq_fixture):
         ).all(), variable
 
 
+def test_steadystate_computation_mode(preeq_fixture):
+    """Test newtonOnly and integrationOnly steady-state computation modes"""
+    (
+        model,
+        solver,
+        edata,
+        edata_preeq,
+        edata_presim,
+        edata_sim,
+        pscales,
+        plists,
+    ) = preeq_fixture
+
+    solver.setSensitivityOrder(amici.SensitivityOrder.first)
+    solver.setSensitivityMethodPreequilibration(
+        amici.SensitivityMethod.forward)
+    solver.setNewtonMaxSteps(10)
+
+    rdatas = {}
+    stst_computation_modes = [
+        amici.SteadyStateComputationMode.integrationOnly,
+        amici.SteadyStateComputationMode.newtonOnly,
+    ]
+    for mode in stst_computation_modes:
+        model.setSteadyStateComputationMode(mode)
+        rdatas[mode] = amici.runAmiciSimulation(model, solver, edata)
+
+        # assert successful simulation
+        assert rdatas[mode]["status"] == amici.AMICI_SUCCESS
+
+    assert np.all(rdatas[amici.SteadyStateComputationMode.integrationOnly][
+                      'preeq_status'][0] == [0, 1, 0]) and \
+           rdatas[amici.SteadyStateComputationMode.integrationOnly][
+                      'preeq_numsteps'][0][0] == 0
+
+    assert np.all(rdatas[amici.SteadyStateComputationMode.newtonOnly][
+                      'preeq_status'][0] == [1, 0, 0]) and \
+           rdatas[amici.SteadyStateComputationMode.newtonOnly][
+                      'preeq_numsteps'][0][0] > 0
+
+    # assert correct results
+    for variable in ["llh", "sllh", "sx0", "sx_ss", "x_ss"]:
+        assert np.isclose(
```

Comment:
Please use [`numpy.testing.assert_allclose`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html#numpy.testing.assert_allclose) for more readable output in case of failure.